src/main/java/org/veupathdb/lib/container/jaxrs/server/middleware/PrometheusFilter.java

```diff
@@ -160,50 +159,26 @@ public void aroundWriteTo(WriterInterceptorContext context) throws IOException,
   }
 
   private void logResponse(String path, int status, String method, boolean isError) {
-    // use trace for metrics requests to avoid log congestion; warn for errors
-    Consumer<Supplier<?>> fn = isError ? LOG::warn : path.equals("metrics") ? LOG::trace : LOG::debug;
-    fn.accept(() -> format(END_FORMAT, method, "/" + path, status));
-  }
+    LoggingEventBuilder logLine;
+    if (isError) {
+      logLine = LOG.atWarn();
+    } else if (path.equals("metrics")) {
+      logLine = LOG.atTrace();
+    } else {
+      logLine = LOG.atDebug();
+    }
 
-  /**
```

Comment:
It previously _was_ implemented, someone somewhere along the line un-implemented it.

Reply:
additionally, when it was un-implemented, the paths used for metrics were swapped out from the literal paths to the path templates, so instead of recording `/foo/123` we're now recording `/foo/{foo-id}`